internal/client/client_test.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import "testing"
+
+func TestGetResourceScope(t *testing.T) {
+	tests := map[string]struct {
+		scope         string
+		wantCluster   bool
+		wantNamespace bool
+		wantErr       bool
```

Comment:
No need for change, just a thought: If you introduce a struct like this
```go
type result struct {
  cluster, namespace, err bool
}
```

you can do all three comparisons at once like so:
```go
if diff := cmp.Diff(test.want, got); diff != "" {
  t.Errorf("getResourceScope(..) returned unexpected result (-want,+got)\n%s", diff)
}
```

Reply:
Neat! I'll keep that in mind for next time